### PR TITLE
Allow Idle Push

### PR DESCRIPTION
**Pull request by SylveonBottle**
no more nopush checks in spawn while ur afk

### CODE REVIEW

File: Fedoraware/TeamFortress2/TeamFortress2/Hooks/Detours/C_TFPlayer_AvoidPlayers.cpp [hunk 1]
```diff
@@ -5,6 +5,15 @@ MAKE_HOOK(C_TFPlayer_AvoidPlayers, g_Pattern.Find(L"client.dll", L"55 8B EC 81 E
...
 	if (!Vars::Misc::NoPush.Value)
 	{
-		Hook.Original<FN>()(ecx, edx, pCmd);
+		{
+			Hook.Original<FN>()(ecx, edx, pCmd);
+		}
+	}
+	else if (Vars::Misc::NoPush.Value && Vars::Misc::NoPushIdle.Value)
+	{
+		if (pCmd->forwardmove == 0 && pCmd->sidemove == 0 && g_EntityCache.GetLocal()->OnSolid())
```
Comment (johnathon-walnut) [inline Fedoraware/TeamFortress2/TeamFortress2/Hooks/Detours/C_TFPlayer_AvoidPlayers.cpp:14]: Maybe do buttons instead?

Comment (SylveonBottle) [reply Fedoraware/TeamFortress2/TeamFortress2/Hooks/Detours/C_TFPlayer_AvoidPlayers.cpp:14]: No clue how + this worked fine for me while testing

Comment (SylveonBottle) [reply Fedoraware/TeamFortress2/TeamFortress2/Hooks/Detours/C_TFPlayer_AvoidPlayers.cpp]: guess this works?

**Comment by johnathon-walnut** [comment]
I actually meant pCmd->buttons & (IN_FORWARD | IN_BACK | IN_MOVERIGHT | IN_MOVELEFT) lol but that's fine anyway fuck people who don't use WASD

**Comment by SylveonBottle** [comment]
Oh so I made it worse LMAO

**Comment by Eniac763** [comment]
bruhh

**Comment by SylveonBottle** [comment]
tbh I just wanted idle nopush because I was getting tired of bunnyhopping and being yeeted 3 feet to the side every time I tapped a teammate's shoulder
and I didn't want to use nopush because I didn't want people pushing me while I was standing still (waiting for a gate to open / being healed at a dispenser)

**Comment by xxmyn** [comment]
mfed ur a cuck.

**Comment by johnathon-walnut** [comment]
Somewhat trueee

**Pull request merged**